ui/src/keyboardMappings.ts

```diff
@@ -120,6 +121,14 @@ export const keys = {
   Space: 0x2c,
   SystemRequest: 0x9a,
   Tab: 0x2b,
+  ControlLeft: 0xe0,
```

Comment:
These all have to be in the keys collection because we need to get their names when we pass them around